src/components/panes/EditCallerPane.jsx

```diff
@@ -74,18 +74,18 @@ export default class EditCallerPane extends PaneBase {
                 onAdd={ this.onAddTag.bind(this, 'ex') }
                 onRemove={ this.onRemoveExTag.bind(this) }/>
         }
-
+        
```

Comment:
This is some redundant whitespace, and we shouldn't introduce any such changes unnecessarily.